src/main/java/org/mule/extension/http/internal/request/HttpRequesterProvider.java

```diff
@@ -104,14 +107,32 @@
   @Placement(tab = AUTHENTICATION)
   private HttpRequestAuthentication authentication;
 
+  @Parameter
+  @Optional
+  @Placement(tab = "Test Connection")
+  protected HttpConnectivityValidator testConnection;
```

Comment:
this should ideally be a `@ParameterGroup(showInDsl = true)`, that is of course unless you have a really good reason to treat this as a pojo

Reply:
If it's a pojo it can both be optional and have mandatory parameters, so it's easy to preserve backwards compatibility: if some old application hasn't the test-connection tab, then it doesn't execute the connectivity test and everything works as before.
If it's a pg, we must make all the parameters optional, because otherwise an app which doesn't define the test-connection won't be able to be deployed